content/en/cd-as-a-service/setup/deploy-your-app.md

```diff
@@ -0,0 +1,231 @@
+    strategy: trafficSplit
+    constraints:
+      dependsOn: ["staging"]
+      beforeDeployment:
+        - pause:
+            untilApproved: true
+manifests:
+  - path: manifests/my-app.yaml
+  - path: manifests/my-app-service.yaml
+  - path: manifests/namespace-staging.yaml
+    targets: ["staging"]
+  - path: manifests/namespace-prod.yaml
+    targets: ["prod"]
+strategies:
+  rolling:
+    canary:
+      steps:
+        - setWeight:
+            weight: 100
+  trafficSplit:
+    canary:
+      steps:
+        - setWeight:
+            weight: 25
+        - exposeServices:
+            services:
+              - <my-app-service>
+            ttl:
+              duration: 30
+              unit: minutes
+        - pause:
+            untilApproved: true
+        - setWeight:
+            weight: 100
+```
+
+* Replace `account` and `namespace` values with those specific to your environment. 
+* Replace `<my-app>` with your app name (or the names of your manifest file)
+
+{{% alert title="Important" color="warning" %}}
+For the first deployment of your app, Armory CD-as-a-Service automatically deploys the app to 100% of the cluster since there is no previous version. Subsequent deployments of your app follow the strategy steps defined in your deployment file.
+{{% /alert %}}
+
+## Deploy your app
+
+{{% alert title="Important" color="warning" %}}
+Armory CD-as-a-Service manages your Kubernetes deployments using ReplicaSet resources. During the initial deployment of your app, CD-as-a-Service deletes the underlying Kubernetes deployment object in a way that leaves behind the ReplicaSet and pods so that there is no actual downtime for your app. These are later deleted when the deployment succeeds.
+
+If your initial deployment fails, you should [manually delete]({{< ref "cd-as-a-service/troubleshooting/tools#initial-deployment-failure-orphaned-replicaset" >}}) the orphaned ReplicaSet.
+{{% /alert %}}
+
+### Deploy first version
+
+1. Log in using the CLI.
+
+   ```bash
+   armory login
+   ```
+
+   The CLI returns a `Device Code` and opens your default browser.  Confirm the code in your browser to complete the login process,.
+
+   After you successfully authenticate, the CLI returns a list of tenants if you have access to more than one. Select the tenant you want to access. Note that most users only have access to one tenant. If you have access to several tenants, you can log in directly to your desired tenant with `armory login -e '<tenant>'`.
+
+1. Start the deployment from the root of your directory.
+
+   ```bash
+   armory deploy start  -f deployment.yaml
+   ```
+
+   The `deploy start` command starts your deployment and progresses until the first pause for manual approval. It also returns a **Deployment ID** that you can use to check the status of your deployment and a link to your deployment's details page in the [CD-as-a-Service Console](https://console.cloud.armory.io/deployments).
```

Comment:
The `deploy start` command starts your deployment and returns a Deployment ID and a link to your deployment. You can navigate to the deployment details page in CD-as-a-Service Console to see your deployment execution. 

The command itself just starts the deployment, and returns ID and URL. The deployment runs according to strategy.

Reply:
Yes... and the strategy used for this guide has a manual approval for prod deployment.  Clarified the content in the _Deploy first version_ section.